script.matchcenter/resources/skins/default/720p/script-matchcenter-Livescores.xml

```diff
@@ -11,17 +11,18 @@
 		<effect type="fade" start="100" end="0" time="240" />
 	</animation>
 	<controls>
-		<control type="image">
+		<control type="image" id="32502">
 			<left>158</left>
 			<top>24</top>
 			<width>966</width>
 			<height>663</height>
+			<visible>IsEmpty(Window(home).Property(loading))</visible>
```

Comment:
would be cool to attach these properties to the script window, or, as an alternative, prefix the property name.